examples/feeds/link_feed_item_set.py

```diff
@@ -0,0 +1,133 @@
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_id: the ID of the feed associated with the specified feed item set.
+        feed_item_id: the ID of the specified feed item.
+        feed_item_set_id: the ID of the feed item set to link to the feed item.
+    """
+    feed_item_set_link_service = client.get_service(
+        "FeedItemSetLinkService", version="v6"
+    )
+    feed_item_set_link_operation = client.get_type(
+        "FeedItemSetLinkOperation", version="v6"
+    )
+
+    # Construct an operation that will link the feed item to the feed item set.
+    feed_item_set_link = feed_item_set_link_operation.create
+
+    # Construct a resource name for a feed item set link, which is in the
+    # format: customers/{customer_id}/feedItemSetLinks/{feed_id}~{feed_item_set_id}~{feed_item_id}
+    feed_item_set_link.feed_item = client.get_service(
+        "FeedItemService", version="v6"
+    ).feed_item_path(
+        customer_id, ResourceName.format_composite(feed_id, feed_item_id),
+    )
+    # Construct a resource name for a feed item set, which is in the
+    # format: customers/{customer_id}/feedItemSets/{feed_id}~{feed_item_set_id}
+    feed_item_set_link.feed_item_set = client.get_service(
+        "FeedItemSetService", version="v6"
+    ).feed_item_set_path(
+        customer_id, ResourceName.format_composite(feed_id, feed_item_set_id),
+    )
+
+    # Issue a mutate request to add the feed item set link on the server.
+    try:
+        response = feed_item_set_link_service.mutate_feed_item_set_links(
+            customer_id, [feed_item_set_link_operation]
+        )
+        print(
+            "Created a feed item set link with resource name: "
+            f"'{response.results[0].resource_name}'"
+        )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+if __name__ == "__main__":
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description=
+            "Links the specified feed item set to the specified feed item."
+    )
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument(
+        "-c",
+        "--customer_id",
+        type=str,
+        required=True,
+        help="The Google Ads customer ID.",
+    )
+    parser.add_argument(
```

Comment:
`black` formatter wanted to put this on one line. FYI @wihl that's where that's coming from

Reply:
Strange that Black is inconsistent here